domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaimData.java

```diff
@@ -22,8 +22,9 @@
 import static uk.gov.hmcts.cmc.domain.models.sampledata.SampleInterest.noInterestBuilder;
 
 public class SampleClaimData {
+    public static final String EXTERNAL_REFERENCE_NUMBER = "CLAIM234324";
 
-    private UUID externalId = UUID.randomUUID();
+    private String externalId = SampleClaim.EXTERNAL_ID;
```

Comment:
Why are we using a less strict type?

Reply:
I had changed it to make it more manageable, but I ended up removing the changes that consumed it directly and indeed in the `ClaimData` class it's still explicitly a `UUID`.

Reverting